cmd/keytransparency-signer/backend.go

```diff
@@ -80,8 +82,11 @@ func newMapServer(ctx context.Context, sqldb *sql.DB, factory ctxn.Factory) (tri
 	if err != nil {
 		return nil, err
 	}
-
-	signer, err := keys.NewFromPrivatePEM(*signingKey, "")
+	pemB, err := ioutil.ReadFile(*signingKey)
+	if err != nil {
+		return nil, err
+	}
+	signer, err := keys.NewFromPrivatePEM(string(pemB), *signingKeyPassword)
```

Comment:
Thanks! I changed it to `NewFromPrivatePEMFile`. Just a note: Now, there is a minor difference in behavior though: with [`NewFromPrivatePEM`](https://github.com/google/trillian/blob/0ddedc05cdabd5539a0d4c869db06c9ca5a29044/crypto/keys/private_keys.go#L74-L90) one could provide an empty password which isn't possible with [`NewFromPrivatePEMFile`](https://github.com/google/trillian/blob/0ddedc05cdabd5539a0d4c869db06c9ca5a29044/crypto/keys/private_keys.go#L57) anymore (and I think that makes more sense, too).